README.md

```diff
@@ -66,6 +66,106 @@ To run all tests in the project, use the following at the command line.
 
 ```green```
 
+### Wishlist Endpoints
+​
+1. **Create Wishlist**
+   - **Endpoint**: `POST /wishlists`
+   - **Description**: Create a new wishlist for a customer.
+   - **Request JSON**:
+     ```json
+     {
+         "customer_id": 1,
+         "wishlist_name": "My Wishlist",
+         "created_date": "2023-10-12"
+     }
+     ```
+   - **Response**:
+     - **Status Code**:
+       - 201 (Created) - Successfully created a new wishlist.
+     - **Body**: Returns the newly created wishlist with its ID.
+​
+2. **Get Wishlist**
+   - **Endpoint**: `GET /wishlists?id=<wishlist_id>`
+   - **Description**: Retrieve information about a specific wishlist.
+   - **Response**:
+     - **Status Codes**:
+       - 200 (OK) - Successfully retrieved the wishlist.
+       - 404 (Not Found) - Wishlist not found.
+​
+3. **Delete Wishlist**
+   - **Endpoint**: `DELETE /wishlists/<wishlist_id>`
+   - **Description**: Delete a specific wishlist.
+   - **Response**:
+     - **Status Codes**:
+       - 204 (No Content) - Successfully deleted the wishlist.
+       - 404 (Not Found) - Wishlist not found.
+​
+4. **List Wishlists**
+   - **Endpoint**: `GET /wishlists`
+   - **Description**: Retrieve a list of all wishlists.
+   - **Response**:
+     - **Status Code**:
+       - 200 (OK) - Successfully retrieved the list of wishlists.
```

Comment:
More specifically, we return the wishlist meta information e.g. id, created_date, customer_id, and wishlist_name.
Rather than the whole wishlist with nested lists.